tools/travis/run-luacheck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing Lua 5.3, LuaRocks and Luacheck"
+(
```

Comment:
Travis persists the cache folder over builds, so Installing just once would be enough. Maybe give it a version number to be able to update to a newer version if needed.
I am not sure if this has to go in this PR, so only a comment.